examples/fscloud/main.tf

```diff
@@ -72,7 +72,8 @@ module "postgresql_db" {
     "postgressql_viewer" : "Viewer",
     "postgressql_editor" : "Editor",
   }
```

Comment:
`redis` and `mongo` have different order `service_credential_names` is after `deletion_protection`